jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/URIUtil.java

```diff
@@ -1622,6 +1465,7 @@ static boolean equalsIgnoreEncodings(String uriA, String uriB)
         }
     }
 
+    // Only used by URIUtil
```

Comment:
Make it private then.

Reply:
Can't make it private, as it's tested in URIUtilTest.
And protected isn't valid.

Reply:
Then the comment is wrong, should be `Only used by URIUtil*Test*.`

Reply:
It's used by URIUtil as a sub-method, and that sub-method is tested in URIUtilTest, it's not exposed to others.

Reply:
Went ahead and cleaned it out